### ISSUE TITLE
Inline preview of cells not in focus

### ISSUE THREAD

**Issue by nschneid** [HappenApps/Quiver#13]
It seems superfluous to have to open a separate view (Preview) to see rendered Markdown, LaTeX, etc. when those cells are not being edited. IPython Notebook follows a similar cell-based model, but renders these sorts of things immediately when the cell loses focus, and reverts to the source form only when the cell regains focus for editing.

**Comment by ylian** [comment]
I tried that when implementing the preview feature. The problem is, the user experience is not good. The cell height changes when rendered, so when you click around the note, you see content move up and down. I have used IPython Notebook, and it has the same issue. For Quiver, it's far more important to make the editing experience as good as possible --- that's why most things are edited in-place and nothing shifts underneath.
